apps/root/src/pages/aggregator/swap-container/components/quote-data/index.tsx

```diff
@@ -74,19 +74,22 @@ const QuoteData = ({ quote, to, isBuyOrder }: QuoteDataProps) => {
                       intl,
                     })} ${quote?.sellToken.symbol}`
                   : '-'}
-                <Tooltip
-                  title={
-                    <FormattedMessage
-                      description="quoteDataMaximumTooltip"
-                      defaultMessage="This is the maximum you will spend based on your slippage settings"
-                    />
-                  }
-                  arrow
-                  placement="top"
-                >
-                  <HelpOutlineIcon color="inherit" fontSize="inherit" />
-                </Tooltip>
               </Typography>
+              <Tooltip
+                title={
+                  <FormattedMessage
+                    description="quoteDataMaximumTooltip"
+                    defaultMessage="This is the maximum you will spend based on your slippage settings"
+                  />
+                }
+              >
+                <ContainerBox>
```

Comment:
do we actually need this container box?

Reply:
Yes, tooltips doesn't work directly over svg icons, I think becuase they are not capable of holding ref's